Squablr/View Controllers/UserFeedViewController.h

```diff
@@ -6,10 +6,15 @@
 //
 
 #import <UIKit/UIKit.h>
+#import "ZLSwipeableView.h"
+#import "CardView.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
-@interface UserFeedViewController : UIViewController
+@interface UserFeedViewController : UIViewController <ZLSwipeableViewDataSource, ZLSwipeableViewDelegate, UIActionSheetDelegate>
+
+@property (nonatomic, strong) ZLSwipeableView *swipeableView;
```

Comment:
I would put a space in between your attribute declarations and method declarations